client/errors.go

```diff
@@ -0,0 +1,9 @@
+package client
+
+import "errors"
+
+var (
+	ErrUnauthorized = errors.New("unauthorized")
```

Comment:
Should we have our own errors here? or are there errors to borrow in http package?
Maybe it is better with our own?

Reply:
I could not find errors in the http package since for them it is not an "error". A 404 call is just another valid response.